admin_manual/configuration/user/user_types.rst

```diff
@@ -0,0 +1,117 @@
+==============
+ownCloud Roles
+==============
+
+ownCloud supports six user types.
+These are:
+
+- `Anonymous`_
+- `Guest`_
+- `Standard User`_
+- `Federated User`_
+- `ownCloud Group Administrator`_
+- `ownCloud Administrator`_
+- `System Administrator`_
+- `Auditor`_
+
+The following information is not an in-depth guide, but more of a high-level overview of each type.
+
+Anonymous
+---------
+
+- Is not a regular user.
+- Has access to specific content made available via public links.
+  - Can be password-protected (optional, enforced, policy-enforced).
+  - Can have an expiration date (optional, enforced, enforced dependent on password).
+- Has no personal space
+- Has no file ownership (ownership of uploaded/created files is directed to sharer).
+- Has no use of clients.
+- Quota is that of the sharer.
+- Permissions are those granted by the sharer for specific content, e.g., *view-only*, *edit*, and *no file listing*.
```

Comment:
Please call "no file listing" = "File Drop". Will also be changed in the UI with 10.0.4 soon.